src/handlers/comment/handlers/query.ts

```diff
@@ -19,7 +19,7 @@ export const query = async (body: string) => {
     return `Skipping '/query' because of no issue instance`;
   }
 
-  const regex = /^\/query\s@(\w+)$/;
+  const regex = /(?<=\/query @)\w+/g
   const matches = body.match(regex);
```

Comment:
`match` function behaves [differently](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/match#return_value) when a regex is global like yours `/g`. I think we don't need global regex because we are just searching for one instance of query